aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -96,206 +127,442 @@ impl AggregatorID {
 ///
 /// TODO: while we support tracking of the history, it is not yet fully used on
 /// executor side because we don't know how to throw errors.
```

Comment:
We need to update the picture above to include the overflow bounds.
We probably don't need the TODO, as we can't avoid having to resolve it as a part of current PRs (and hard to remember to remove the TODO). It'd make sense to keep it if it'd be unaddressed.